buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -54,7 +54,7 @@ object Versions {
   const val playCore = "1.10.0"
   const val fastScroll = "2.0.0"
   const val mixpanel = "5.6.4"
-  const val mobius = "1.3.4"
+  const val mobius = "1.5.3"
```

Comment:
Was changing the version twice approach to make sure there are no major changes in the versions and nothing is broke?

Reply:
Yep